client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MlClientDocumentationIT.java

```diff
@@ -2935,8 +2940,20 @@ public void testPutDataFrameAnalytics() throws Exception {
                 .build();
             // end::put-data-frame-analytics-outlier-detection-customized
 
+            // tag::put-data-frame-analytics-classification
+            DataFrameAnalysis classification = org.elasticsearch.client.ml.dataframe.Classification.builder("my_dependent_variable") // <1>
+                .setLambda(1.0) // <2>
+                .setGamma(5.5) // <3>
+                .setEta(5.5) // <4>
+                .setMaximumNumberTrees(50) // <5>
+                .setFeatureBagFraction(0.4) // <6>
+                .setPredictionFieldName("my_prediction_field_name") // <7>
+                .setTrainingPercent(50.0) // <8>
+                .build();
+            // end::put-data-frame-analytics-classification
+
             // tag::put-data-frame-analytics-regression
-            DataFrameAnalysis regression = Regression.builder("my_dependent_variable") // <1>
+            DataFrameAnalysis regression = org.elasticsearch.client.ml.dataframe.Regression.builder("my_dependent_variable") // <1>
```

Comment:
We have two classes named `Regression` (similarly with `Classification`): one from `analyses` package, the other one from `evaluation` package.
We didn't have this problem with outlier detection as the analysis is `OutlierDetection` and its evaluation is `BinarySoftClassification`.
What I could do here to avoid fully-qualified names is renaming, say, `Regression` from evaluation package to `RegressionEvaluation`. 
WDYT?

Reply:
I think it's generally nice to avoid duplicate class names for multiple classes that users are likely to want to use within a single source file.  So I like the idea of renaming `Regression` to `RegressionEvaluation` if we think users will want to use both in the same client program source file.  Maybe they won't though and this docs page is the unusual case of using both in the same source file.  Even if a rename would be justified I'm happy to do the rename in a separate PR to allow the docs to move forward.  Fully qualifying both where they're used in the same docs page is fine for now.